README.md

```diff
@@ -62,7 +62,7 @@ user: generator, password: hitmeup
 ```
 cat > /tmp/case_01.json <<EOF
 {
-    "eventType": "CASE_CREATED",
+    "eventType": "CASE_UPDATE",
     "source": "RESPONDENT_HOME",
     "channel": "RH",
     "contexts": [
```

Comment:
Line 92: add a gap between the markup hashes and **Response** since it doesn't render properly